ettersendelse/src/main/java/no/nav/k9/ettersendelse/Ettersendelse.java

```diff
@@ -32,36 +32,47 @@ public class Ettersendelse implements Innsending {
     @Valid
     private final Versjon versjon;
 
-
     @JsonProperty(value="mottattDato")
     @Valid
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSX", timezone = "UTC")
     private final ZonedDateTime mottattDato;
 
-
     @JsonProperty(value="søker", required = true)
     @Valid
     @NotNull
     private final Søker søker;
 
-
     @JsonProperty(value="ytelse", required = true)
     @Valid
     @NotNull
     private final Ytelse ytelse;
 
+    @JsonProperty(value="pleietrengende")
+    @Valid
+    @NotNull
```

Comment:
Brukes denne kontrakten i dag? I så fall burde den kanskje ikkje settes til NotNull. Dersom vi tenkjer å ta i bruk for omsorgspenger så er vel ikkje pleietrengende relevant der heller.

Reply:
godt poeng!